dubbo-common/src/main/java/org/apache/dubbo/common/utils/PojoUtils.java

```diff
@@ -541,6 +549,26 @@ private static Object newInstance(Class<?> cls) {
         }
     }
 
+    /**
+     * return init value when type is primitive
+     * @param parameterType
+     * @return
+     */
+    private static Object getPrimitiveDefaultValue(Class<?> parameterType) {
+        if (parameterType.getName().equals("int") || parameterType.getName().equals("short")

```

Comment:
java.lang.Class provides a method call isPrimitive(). I think it can be used here:

```
if (parameterType is char) return Character.MINVALUE;
if (parameterType is primitive) return 0;
return null
```

Does that look simpler?

Reply:
yes , make this method become simple and general